src/remoteSync/s3/validator.js

```diff
@@ -0,0 +1,27 @@
+const files = require('./../../utils/files');
+
+const s3ConfigKeys = {
+    s3AccKeyLoc: 's3sdkPath',
+};
+
+const validateInitConfig = async (config) => {
+    let validatedConfig = {};
+
+    if (!config[s3ConfigKeys.s3AccKeyLoc]) {
```

Comment:
Add docs for configuring using file:
https://github.com/hariprasanths/synchly/blob/master/docs/configuration-using-file.md#configuration-using-file

You can test it using this command: synchly --config=remote-sync --file=/home/foo/config.json